src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -194,61 +194,65 @@ protected override void BeginProcessing()
         protected override void ProcessRecord()
         {
             bool result = true;
-            string resolvedPath = string.Empty;
             string jsonToParse = string.Empty;
 
             if (Json != null)
             {
                 jsonToParse = Json;
             }
 
```

Comment:
Please remove this blank line.

Reply:
@daxian-dbw I just removed the `resolvedPath` in most recent feedback commit https://github.com/PowerShell/PowerShell/pull/19042/commits/0aaf07c324683f5b901e308e70c3a9ca7e556fdf. In the actual whole diff there is no blank line.

Reply:
Ah I see a blank line on line 168 now, will remove that.